.cirrus.yml

```diff
@@ -652,6 +652,15 @@ task:
         - ulimit -S -n 2048 # https://github.com/flutter/flutter/issues/2976
         - ./dev/bots/deploy_gallery.sh
 
+    - name: verify_binaries_codesigned-macos # macos-only
+      # TODO(fujino): remove this `only_if` after https://github.com/flutter/flutter/issues/44372
+      only_if: "$CIRRUS_BRANCH == 'dev' || $CIRRUS_BRANCH == 'beta' || $CIRRUS_BRANCH == 'stable'"
```

Comment:
Late add - we should make this run on hotfix branches too.

Reply:
Oh yeah, that'd be nice. Can you do pattern matching in the `.cirrus.yml` file @fkorotkov?

Reply:
[Yes](https://cirrus-ci.org/guide/writing-tasks/#supported-operators), you can do something like `$CIRRUS_BRANCH =~ '.*hotfix.*'`

Reply:
@fkorotkov thanks!